### PR TITLE
Ct/load institution db

### PR DESCRIPTION
**Pull request by Craiting**
Loading the institution json file into the database.

Also adds `institution_provider_id` because it's an essential field that was not included by mistake.

### CODE REVIEW

File: apps/institution-service/tasks/loadInstitutionsFromJson.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import { Institution } from "../models/institution";
+import { Provider } from "../models/provider";
+
+export interface CachedInstitution {
+  name: string;
+  keywords: string | null;
+  logo: string;
+  url: string;
+  ucp_id: string;
+  is_test_bank: boolean | false;
```
Comment (tphalp) [inline apps/institution-service/tasks/loadInstitutionsFromJson.ts:13]: I've never seen `boolean | false` before. What does this do?

Comment (tphalp) [reply apps/institution-service/tasks/loadInstitutionsFromJson.ts:13]: I was thinking maybe it's a way to set a default value, but I can't find any documentation to support that. I don't think using `boolean | false` is necessary here. `boolean` should suffice, unless I'm not understanding the meaning here.

**Review by tphalp — COMMENTED** [review]
I left a comment.

File: apps/institution-service/tasks/loadInstitutionsFromJson.ts (hunk 1 above)
Comment (Craiting) [reply apps/institution-service/tasks/loadInstitutionsFromJson.ts:13]: Yeah that was an accident, it was supposed to be boolean | null. I'll fix it.

Comment (Craiting) [reply apps/institution-service/tasks/loadInstitutionsFromJson.ts]: Actually I don't even need the `| null` at all anymore.

**Review by tphalp — APPROVED** [review]

**Pull request merged**